README.md

```diff
@@ -39,11 +39,9 @@ Trellis will configure a server with the following and more:
 ## Installation
 
 1. Download/fork/clone this repo to your local machine.
-2. Run `ansible-galaxy install -r requirements.yml` to install external Ansible roles/packages.
+2. Install external roles/packages: run `ansible-galaxy install -r requirements.yml` (in Trellis dir)
```

Comment:
Don't agree with this. The wording is pretty clunky.

Run `ansible-galaxy install -r requirements.yml` inside your Trellis directory to install external Ansible roles/packages.